cmd/test.go

```diff
@@ -201,11 +204,21 @@ func opaTest(args []string) int {
 		capabilities = ast.CapabilitiesForThisVersion()
 	}
 
+	var schemaSet *ast.SchemaSet
+	//	-s {file} (one input schema file)
+	//	-s {directory} (one schema directory with input and data schema files)
+	schemaSet, err = loader.Schemas(testParams.schema.path)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		return 1
+	}
+
 	compiler := ast.NewCompiler().
 		SetErrorLimit(testParams.errLimit).
 		WithPathConflictsCheck(storage.NonEmpty(ctx, store, txn)).
 		WithEnablePrintStatements(!testParams.benchmark).
-		WithCapabilities(capabilities)
+		WithCapabilities(capabilities).
+		WithSchemas(schemaSet)
```

Comment:
Likewise, to make the parser pick up those annotations, `WithProcessAnnotation` must be set on the file loaders at https://github.com/open-policy-agent/opa/blob/dfe947a74655bc902e84bbb8d40946237a8576a8/tester/runner.go#L591 and https://github.com/open-policy-agent/opa/blob/dfe947a74655bc902e84bbb8d40946237a8576a8/tester/runner.go#L619

Reply:
Hi @johanfylling, thanks for the comments.

Yes, I identified the lack of annotation support and I mentioned that on the [issue 5923](https://github.com/open-policy-agent/opa/issues/5923) and was wondering if that should be another issue or not (since technically they are different features). What do you think? Work on that on a different issue or on this one?

Reply:
Sorry @renatosc , I missed your first comment. If I'm to express an opinion here, I think that to not break user expectations with this feature, it's probably a good idea to have `test` work similarly to `eval` and `check` when using the `--schema` flag.